### PR TITLE
draft docs for web modeler api tutorial

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Drafted docs for Web Modeler API tutorial.

**NOTE: In `draft` as this may change based on final review in https://github.com/camunda/camunda-api-tutorials/pull/5. Will backport once finalized and reviewed.**

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/apis-tools/web-modeler-api/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+sidebar_position: 3
+slug: /apis-tools/web-modeler-api/tutorial
+description: "Step through an example to view your existing clients, create a client, view a particular client's details, and delete a client."
+---
+
+In this tutorial, we'll step through examples to highlight the capabilities of the Web Modeler API, such as creating and deleting a file.
+
+## Prerequisites
+
+- If you haven't done so already, [create a cluster](/components/react-components/create-cluster.md).
```
Comment (akeller) [inline docs/apis-tools/web-modeler-api/tutorial.md:13]: Do you actually need a cluster? I don't think you do anything with it in this tutorial, do you?

File: docs/apis-tools/web-modeler-api/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
+---
+id: modeler-api-tutorial
+title: Tutorial
+sidebar_position: 3
+slug: /apis-tools/web-modeler-api/tutorial
+description: "Step through an example to view your existing clients, create a client, view a particular client's details, and delete a client."
```
Comment (akeller) [inline docs/apis-tools/web-modeler-api/tutorial.md:6]: This is the Admin API description - oops!

File: docs/apis-tools/web-modeler-api/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+:::note
+Make sure you keep the generated client credentials in a safe place. The **Client secret** will not be shown again. For your convenience, you can also download the client information to your computer.
+:::
+
+- In this tutorial, we utilize a JavaScript-written [GitHub repository](https://github.com/camunda/camunda-api-tutorials) to write and run requests. Clone this repo before getting started.
+- Ensure you have [Node.js](https://nodejs.org/en/download) installed as this will be used for methods that can be called by the CLI (outlined later in this guide). Run `npm install` to ensure you have updated dependencies.
+
+## Getting started
+
+- A detailed API description can be found [here](https://modeler.cloud.camunda.io/swagger-ui/index.html) via Swagger. With a valid access token, this offers an interactive API experience against your Camunda 8 cluster.
```
Comment (akeller) [inline docs/apis-tools/web-modeler-api/tutorial.md:25]: > With a valid access token, this offers an interactive API experience against your Camunda 8 cluster.

I would reword this since this API isn't scoped for the cluster level.

File: docs/apis-tools/web-modeler-api/tutorial.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+
+:::note
+This `create` command is connected to the `createFile` function at the bottom of the `modeler.js` file, and executed by the `cli.js` file. While we create and delete a file in this tutorial, you may add additional arguments depending on the API calls you would like to make.
+:::
+
+If you have a file matching your file ID, the `Name: {name}; ID: {id} ` will now output. If you have an invalid API name or action name, or no arguments provided, or improper/insufficient credentials configured, an error message will output as outlined in the `cli.js` file.
+
+## DELETE a client
+
+To delete a client, take the following steps:
```
Comment (akeller) [inline docs/apis-tools/web-modeler-api/tutorial.md:120]:
```suggestion
## DELETE a file

To delete a file, take the following steps:
```

File: docs/apis-tools/web-modeler-api/tutorial.md (hunk 1 above)
Comment (akeller) [reply docs/apis-tools/web-modeler-api/tutorial.md:13]: Reopening this - I need to see where a cluster would be necessary in this tutorial. Can you explain why you would need a cluster?

File: docs/apis-tools/web-modeler-api/tutorial.md [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+slug: /apis-tools/web-modeler-api/tutorial
+description: "Step through an example to create a new project, add a collaborator, view project details, and delete a project using the Web Modeler API."
+---
+
+In this tutorial, we'll step through examples to highlight the capabilities of the Web Modeler API, such as creating a new project, adding a collaborator to a project, viewing the details of a project, and deleting a project.
+
+## Prerequisites
+
+- If you haven't done so already, [create a cluster](/components/react-components/create-cluster.md).
+- Upon cluster creation, create your first client by navigating to **Console > Organization > Administration API > Create new credentials**. Ensure you determine the scoped access for client credentials. For example, in this tutorial we will create a project, add a collaborator, and delete a project. Ensure you check all the boxes for Web Modeler and Zeebe client scopes.
```
Comment (akeller) [inline docs/apis-tools/web-modeler-api/tutorial.md:14]: Does this tutorial really require Zeebe client scopes?

File: docs/apis-tools/web-modeler-api/tutorial.md [hunk 6]
```diff
@@ -0,0 +1,247 @@
...
+
+```javascript
+const authorizationConfiguration = {
+  clientId: process.env.MODELER_CLIENT_ID,
+  clientSecret: process.env.MODELER_CLIENT_SECRET,
+  audience: process.env.MODELER_AUDIENCE,
+};
+```
+
+2. Examine the function `async function createProject([projectName, adminEmail])` below this configuration. This is where you will script out your API call, defining a project name and the admin's email.
```
Comment (akeller) [inline docs/apis-tools/web-modeler-api/tutorial.md:60]: This might be a nitpick, but I assumed the admin email had something to do with the administration API. After reading through the tutorial and looking at Swagger, I realized this is the project admin.

File: docs/apis-tools/web-modeler-api/tutorial.md [hunk 7]
```diff
@@ -0,0 +1,247 @@
...
+3. Within the function, you must first apply an access token for this request, so your function should now look like the following:
```
Comment (pepopowitz) [inline docs/apis-tools/web-modeler-api/tutorial.md:61]:
```suggestion
3. Within the function, you must first generate an access token for this request, so your function should now look like the following:
```

non-blocking nit, this feels like a more accurate verb to me because we aren't applying the token to anything until a few more steps.

File: docs/apis-tools/web-modeler-api/tutorial.md [hunk 8]
```diff
@@ -0,0 +1,247 @@
...
+```javascript
+async function createProject([projectName, adminEmail]) {
+  const accessToken = await getAccessToken(authorizationConfiguration);
+}
+```
+
+4. Using your generated client credentials from [prerequisites](#prerequisites), capture your Web Modeler base URL beneath your call for an access token by defining `ModelerBaseUrl`:
+
+```javascript
+const ModelerApiUrl = process.env.MODELER_BASE_URL;
```
Comment (pepopowitz) [inline docs/apis-tools/web-modeler-api/tutorial.md:72]:
```suggestion
4. Using your generated client credentials from [prerequisites](#prerequisites), capture your Web Modeler base URL beneath your call for an access token by defining `modelerApiUrl`:

```javascript
const modelerApiUrl = process.env.MODELER_BASE_URL;
```

this is what we named it in the companion repo.

File: docs/apis-tools/web-modeler-api/tutorial.md [hunk 9]
```diff
@@ -0,0 +1,247 @@
...
+  const response = await axios(options);
+  const project = response.data;
+
+  console.log("Project:", project);
+} catch (error) {
+  console.error(error.message);
+}
+```
+
+4. In your terminal, run `npm run cli modeler view <project ID>`, where `<project ID>` is where you can paste the ID of the project you would like to view.
```
Comment (pepopowitz) [inline docs/apis-tools/web-modeler-api/tutorial.md:195]:
```suggestion
4. In your terminal, run `npm run cli modeler view <project ID>`, where `<project ID>` is the ID output by the command to create a project.
```

We can give them a more specific ID, so they aren't wondering where to get one.

File: docs/apis-tools/web-modeler-api/tutorial.md [hunk 10]
```diff
@@ -0,0 +1,247 @@
...
+    console.log(`Project ${projectId} was deleted!`);
+  } else {
+    console.error("Unable to delete project!");
+  }
+} catch (error) {
+  console.error(error.message);
+}
+```
+
+4. In your terminal, run `npm run cli modeler delete <project ID>`, where `<project ID>` is where you can paste the ID of the project you would like to delete.
```
Comment (pepopowitz) [inline docs/apis-tools/web-modeler-api/tutorial.md:239]:
```suggestion
4. In your terminal, run `npm run cli modeler delete <project ID>`, where `<project ID>` is the ID output by the command to create a project.
```

**Review by pepopowitz — COMMENTED** [review]
A few small changes from my end, but this is close!

File: docs/apis-tools/web-modeler-api/tutorial.md (hunk 1 above)
Comment (christinaausley) [reply docs/apis-tools/web-modeler-api/tutorial.md:13]: Adjusted in latest commit 👍

File: docs/apis-tools/web-modeler-api/tutorial.md (hunk 5 above)
Comment (christinaausley) [reply docs/apis-tools/web-modeler-api/tutorial.md:14]: Adjusted in latest commit 👍

File: docs/apis-tools/web-modeler-api/tutorial.md (hunk 7 above)
Comment (christinaausley) [reply docs/apis-tools/web-modeler-api/tutorial.md:61]: Adjusted in latest commit 👍

File: docs/apis-tools/web-modeler-api/tutorial.md (hunk 8 above)
Comment (christinaausley) [reply docs/apis-tools/web-modeler-api/tutorial.md:72]: Adjusted in latest commit 👍

File: docs/apis-tools/web-modeler-api/tutorial.md (hunk 10 above)
Comment (christinaausley) [reply docs/apis-tools/web-modeler-api/tutorial.md:239]: Resolved in latest commit 👍

**Comment by christinaausley** [comment]
@pepopowitz @akeller I believe this is ready for final review 👍

File: docs/apis-tools/web-modeler-api/tutorial.md (hunk 6 above)
Comment (christinaausley) [reply docs/apis-tools/web-modeler-api/tutorial.md]: I adjusted the wording a bit, but let me know if this doesn't resolve your concern.

**Review by akeller — APPROVED** [review]
Thanks for incorporating my feedback!

**Review by pepopowitz — APPROVED** [review]
😍 😍 😍
